pom.xml

```diff
@@ -35,15 +40,38 @@
     <dependency>
       <groupId>com.esri.geoevent.sdk</groupId>
       <artifactId>geoevent-sdk</artifactId>
-      <version>${project.version}</version>
+      <version>[{geoevent.compat.version},)</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.activemq</groupId>
       <artifactId>activemq-client</artifactId>
       <version>${activemq.version}</version>
     </dependency>
-    <!-- dependencies for embedding via the vm and peer transports omitted -->
+    <!-- dependencies for embedding via the vm and peer transports included -->
```

Comment:
So, in addition to functioning as a client to an ActiveMQ broker listening elsewhere, the ActiveMQ client also includes the ability run a broker within the calling process, just by using special URIs. I have tested this and it does work. However, if the required dependencies are not available, it fails.

For more info see here https://activemq.apache.org/activemq-connection-uris under the headings for VM transport, Peer transport, and Broker Configuration URIs.

If you don't want this functionality in the published version of ActiveMQ for GeoEvent, I'm OK with that - I am not using that functionality in production, and I can keep those extra dependency references in a branch that I maintain separately instead, in case needed for tests or demos.

Reply:
note this section is deleted with the second commit; it is not required for normal usage and reduces the JAR size by 3MB. i placed the embedded broker dependencies in a separate branch if would like to talk about it separately, but for this PR that should resolve the ?